site/public/js/server.js

```diff
@@ -306,6 +306,20 @@ function newEditCourseMaterialsForm(path, dir, this_file_section, this_hide_from
         $("#all-sections-showing-yes", form).prop('checked',false);
         $("#all-sections-showing-no", form).prop('checked',true);
     }
+    if (is_link === "1") {
```

Comment:
Is there a reason to use `"1"` over `"true"`? Would `"true"` be more readable?

Reply:
That would be because the value is actually "1" and not "true".